### PR TITLE
Changes to all files

### PR DESCRIPTION
**Pull request by adibalaram**
I've (1) made minor changes to both households.py and SS.py.

(2) Changed everything in firm.py back to what we had done in class since we're not including labor augmenting technological change. Made necessary changes in SS.py too.

(3) Changed all the functions in aggregates.py to reflect Equations (8.18) - (8.21). I'm not entirely certain if the indexing is right in this script.

(4) Checked the length of two vectors that are provided by demographics.py. They are both what was expected. The `imm_rates_SS` variable doesn't exist since `imm_rates_path` is a scalar but someone is trying to index this as if it were a vector (@LeiliPR I think you'd worked on that, so could you please fix it/let us know what you were trying to do).

### CODE REVIEW

**Comment by LeiliPR** [comment]
I have students now, they have exam tomorrow so I cant work until then. I will let you now as soon as I start to work on this again.

**Comment by jdebacker** [comment]
@adibalaram Note that `imm_rates_SS` should be a vector of length `S`.

File: scripts/aggregates.py [hunk 1]
```diff
@@ -4,52 +4,49 @@
...
     '''
-    b = (S-1,) vector of savings
+    Returns total goods consumed (a scalar)
     '''
-    (g_n_SS, omega_SS, imm_rates_SS) = params
-    K = ((1 / (1 + g_n_SS)) * (omega_SS[:-1] * b + imm_rates_SS[1:] *
-                               omega_SS[1:] * b).sum())
 
-    return K
+    Y = C + I - (imm_rates_SS[1:] * omega_SS[1:] * b_s).sum()
```
Comment (jtregde) [inline scripts/aggregates.py:45]: You have `imm_rates_SS[1:]` here, should this just be `imm_rates_SS` if it is a scalar?

Comment (jtregde) [reply scripts/aggregates.py:45]: Based on the above comment, disregard this.

**Comment by adibalaram** [comment]
The variable was obtained using `imm_rates_SS = imm_rates_path[-1, :]`. imm_rates_path is itself a scalar. Since Leili looked into this earlier, I will wait till she gets back to us and one of us can then make the necessary changes.

**Comment by jtregde** [comment]
I think this is because we should use `get_imm_resid()` instead of `get_pop_objs()` here, but yeah, I'll let Leili review it.

**Comment by jdebacker** [comment]
@adibalaram I'd like to merge this PR, which has lots of changes.  But it now conflicts with the master after I merged an earlier PR.  Can you fix those conflicts (all are in `aggregates.py`, I think) and then ping me when this PR is ready to go.  Thanks!

**Comment by adibalaram** [comment]
@jdebacker  The conflicts are in SS.py and households.py from what I see on this page. The files currently in the master branch don't include all the necessary parameters and that's what I had added. I just tried pushing my files again but I'm unable to create a pull request. Do you have any suggestions regarding what I should do next?

**Comment by adibalaram** [comment]
@jdebacker I've fixed all conflicts

File: scripts/households.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
+'''
+Can someone please take a look at this function and make sure that this accounts for everything in the household's problem?
```
Comment (saharnazb) [inline scripts/households.py:2]: I cannot read long lines. Please wrap them.

File: scripts/households.py [hunk 3]
```diff
@@ -18,6 +25,7 @@ def FOCs(b_sp1, n_s, *args):
     l_tilde: maximum amount of labor supply
     chi: scale parameter
     theta: Frisch elasticity of labor supply
+    rho_s: risk that someone alive at age-s will die at the end of that period and not be alive for age s+1
     
     n is not contained in the remaining arguments anymore. If someone decides to change this, please provide detailed documentation
```
Comment (saharnazb) [inline scripts/households.py:35]: The same, please. Wrap this one too. Make it two separate lines

**Comment by jdebacker** [comment]
@adibalaram I'm going to merge this now.  Thanks for updating.

@saharnazb Please open another PR to fix line wraps if you wish.  But you are right - people should wrap to help everyone else read more easily.  Try to keep <=79 characters per line.

**Pull request merged**